components/06-examples/checken.hbs

```diff
@@ -1,27 +1,30 @@
 {{> '@header' }}
 
 <main>
-  <h1>Checken</h1>
   <div class="row">
-    <div class="col-md-9">
-      <h2>Werkzaamheden checken</h2>
-      <h3>Bouwen van een dakkapel</h3>
-
-      {{render '@highlight-box' }}
+    <div class="col-xs-12">
+      <h1>Vergunningcheck</h1>
+        {{render '@steps'}}
     </div>
-    <div class="col-md-3">
-      {{#progressBlock }}
-        <h2>Tussenresultaten</h2>
-        <ul class="list-unstyled">
-          {{#each checkSideListItems }}
-            <li>
-              {{ this }}
-            </li>
-          {{/each}}
-        </ul>
-      {{/progressBlock}}
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      {{#accordion}}
+        {{#each sections }}
+          {{#accordionSection title }}
+            {{> '@form' }}
```

Comment:
Dit werkt prima, maar het kan fraaier: In de yaml configuratie zet je alle formulier properties op een eigen property (bv. `form`):

```yml
sections:
  - title: Plaats van het bouwwerk
    id: panel3
    open: true
    form:
      formLegend: Aanvraagformulier
      fieldsets:
        - legend: Plaats van het bouwwerk
          groups:
          ## [..]
```

Vervolgens geef je het form property mee aan de partial:
```handlebars
{{> '@form' form }}
```

hoeft niet, mag

Reply:
Deze gaan ik nu niet aanpassen. Mogelijk in de toekomst wel, bram is de formulieren bepaalde principes onder de loep aan het nemen. Als die bekend is kan ik hier een aanpassing op doen. Maak ik ook meteen de content meer waarheidsgetrouw.